### DISCUSSION TITLE
Any plans to add Bubble chart?

### DISCUSSION THREAD

**Discussion by sanjoydey22** [airbnb/visx#1587] [General]
Hi Team,
Is there Bubble chart available with visx library? I want to draw a bubble chart like below,
![image](https://user-images.githubusercontent.com/109575982/196887448-d1aeb6de-4745-4954-9af4-ed447a142fe0.png)

It should be greyed out when no data.

**Comment by williaster** [comment]
Hi @sanjoydey22 👋  thanks for checking out visx. I'm converting this to a discussion as it's already supported. 

visx is a **low-level** component library, which means it's more like `d3` than `highcharts`. To create a bubble chart you could use
- `@visx/scale` – position, scale, and/or color circles based on data attributes, you can render these with native `<svg />` and `circle />` elements and control aesthetics as you like
  - `@visx/responsive` could help you make this responsive to screen size
- `@visx/xychart` – this is the only "high-level" chart library in `visx`, it has a `GlpyhSeries` which allows you to render whatever shape (e.g., a circle) you want.
